koji_wrapper/wrapper.py

```diff
@@ -62,9 +65,16 @@ def srpm_url(self, nvr=None):
             # errors here.
             raise inst
 
-    def _build_srpm_url(rpm=None, build=None):
+    def _build_srpm_url(self, rpm=None, build=None):
+        try:
```

Comment:
Not the cleanest way of doing this but and there is a bug here as well

Reply:
As a heads up, self._pathinfo being defined outside of an __init__ method is a linting warning.

If you move it to __init__, you should not need the AttributeError exception handling below.

Reply:
that's probably a better approach, will add an __init__ function in here then.

Reply:
fixed in latest revision of this pull request